Patches/Spaceworm/Spaceworm.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+		<li>Space Worms (Continued)</li>
+		</mods>
+
+		<match Class="PatchOperationSequence">
+			<operations>
+
+			<li Class="PatchOperationAddModExtension">
+				<xpath>/Defs/ThingDef[defName="FaceTeeth"]</xpath>

```

Comment:
FaceTeeth is the defName of the BodyPartGroupDef. The defName for the creature is Scuttlebug.